### PR TITLE
Adding a File for Mumbai

### LINKED ISSUE #4: chandreshsharma's things to be done
- [x] Create an issue.
- [x] Create a branch.
- [x] Create Commits.
- [x] Create a PR.
- [ ] Collaborate
- [ ] Merge

### PR DESCRIPTION
**Pull request by chandreshsharma**
This pull request adds a file with things to do and places to visit in Mumbai. This PR resolves #4.

### CODE REVIEW

File: chandreshsharma-mumbai.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## Mumbai, Maharashtra, India.
+
+### Great Places to eat
```
Comment (chandreshsharma) [inline chandreshsharma-mumbai.md]: Place 1. Juhu Beach

File: chandreshsharma-mumbai.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+### Fun things to do
```
Comment (chandreshsharma) [inline chandreshsharma-mumbai.md]: 1. Visit Esselworld

File: chandreshsharma-mumbai.md (hunk 1 above)
Comment (brentd-smith) [inline chandreshsharma-mumbai.md]: @chandreshsharma Juhu Beach sounds like a great place. Are there mountains nearby? Maybe a boardwalk with a midway (carnival rides)?

**Comment by midatana** [comment]
## test comment to Chandresh' changes

1. good to see all the places you.. #117

File: chandreshsharma-mumbai.md (hunk 1 above)
Comment (chandreshsharma) [inline chandreshsharma-mumbai.md]: Thanks @brentd-smith yes Juhu Beach is great for long walks, especially in the evenings.
No Mountains though, but very beautiful sunsets.

**Pull request merged**